core/trino-main/src/main/java/io/trino/cost/FilterProjectAggregationStatsRule.java

```diff
@@ -70,7 +68,7 @@ protected Optional<PlanNodeStatsEstimate> doCalculate(FilterNode node, StatsProv
             if (!projectNode.isIdentity()) {
                 return Optional.empty();
             }
-            PlanNode projectNodeSource = resolveGroup(lookup, projectNode.getSource());
+            PlanNode projectNodeSource = lookup.resolve(projectNode.getSource());
```

Comment:
Separate this from "Replace invalid resolveGroup usage with resolve" commit.

This place wasn't incorrect. 
it was actually correct, and just working around the (premature) deprecation of `resolveGroup`, doing same logic as `resolve`, but with a local (non deprecated) helper method

Reply:
Moved to "Deduplicate Lookup.resolve copies" commit